src/NzbDrone.Core/Indexers/Definitions/ImmortalSeed.cs

```diff
@@ -171,12 +171,20 @@ private IEnumerable<IndexerRequest> GetPagedRequests(string term, int[] categori
         {
             var searchUrl = Settings.BaseUrl + "browse.php";
 
-            //TODO - Actually map some categories here
             if (term.IsNotNullOrWhiteSpace())
             {
                 searchUrl += string.Format("?do=search&keywords={0}&search_type=t_name&category=0&include_dead_torrents=no", WebUtility.UrlEncode(term));
             }
 
+            searchUrl += "&selectedcats2=";
+
+            foreach (var cat in Capabilities.Categories.MapTorznabCapsToTrackers(categories))
+            {
+               searchUrl += cat + "%2C";
+            }
+
+            searchUrl = searchUrl.Substring(0, searchUrl.Length - 3);
```

Comment:
We could probably cleanup this block a bit using a join to avoid having to cleanup afterwards

Reply:
Try this

```
if (categories != null && categories.Length > 0)
{
    searchUrl += "&selectedcats2=" + string.Join(",", Capabilities.Categories.MapTorznabCapsToTrackers(categories));
}
```